Tests/test_SeqRecord.py

```diff
@@ -128,6 +128,10 @@ def test_valid_features(self):
         with self.assertRaises(TypeError):
             SeqRecord(Seq("ACGT"), features={})
 
+    def test_deprecated_string_seq(self):
+        record = SeqRecord("ACGT")
```

Comment:
That probably should be wrapped to silence (or even check) the expected deprecation warning.

Other than that, no objections @mdehoon?

Reply:
Sure! I will add the improvements to the test.